docs/punctuation/slashes.md

```diff
@@ -1,33 +1,24 @@
----
-title: Slashes - Microsoft Style Guide
-author: pallep
-ms.author: pallep
-ms.date: 09/5/2019
-ms.topic: article
-ms.prod: non-product-specific
----
-
 # Slashes
 
 When necessary, differentiate between a *slash* for URLs and a *backslash* for servers and folders.
 
-When you instruct customers to enter a slash, always include the spelled-out term (*backslash* or *slash*) first, followed by the symbol in parentheses.<br />**Example** <br />Enter two backslashes (\\\\) ....
-
-**See also**<br />[Numbers](~/numbers.md)<br />[Special character term collection](~/a-z-word-list-term-collections/term-collections/special-characters.md)<br />[URLs and web addresses](~/urls-web-addresses.md)
-
-Use a slash:
-
-  - To imply a combination. Capitalize the word after the slash if the word before the slash is capitalized. For example, if *country/region* is used as a label in a form, capitalize it as *Country/Region.*<br />**Examples**<br />client/server<br />TCP/IP<br />CD/DVD drive<br />Use the on/off switch to turn your mouse off when you're not using it.<br />Turn on the On/Off toggle.
+When you instruct customers to enter a slash, always include the spelled-out term (*backslash* or *slash*) first, followed by the symbol in parentheses.
 
-  - To separate parts of an internet address. Use two slashes after the protocol name. <br />**Example** <br />ftp://example.com/downloads
+## Use a slash
 
-  - In server, folder, and file names.<br />**Example** <br />\\\\mslibrary\\catalog\\collect.doc
+Use a slash in the following situations.
 
-  - Between the numerator and denominator of fractions in equations that occur in text.<br />**Examples**<br />a/x + b/y = 1<br />x + 2/3(y) = m
+| Situation | Examples |
+|-----------|----------|
+| **To imply a combination**</br> Capitalize the word after the slash if the word before the slash is capitalized. | client/server</br> TCP/IP</br> Use the on/off switch. |
+| **To separate parts of an internet address**</br> Use two slashes after the protocol name. | `ftp://example.com/downloads` |
+| **In server, folder, and file names** | `\\\\mslibrary\\catalog\\collect.doc` |
```

Comment:
We should use only forward slashes everywhere. `/` is now accepted on Windows as well.